benchexec/tablegenerator/react-table/package.json

```diff
@@ -19,9 +19,41 @@
     "react-vis": "^1.11.7"
   },
   "scripts": {
+    "dependencies": "node scripts/dependencies.js",
     "start": "test -f src/data/dependencies.json || node scripts/dependencies.js && node scripts/start.js",
     "build": "node scripts/dependencies.js && node scripts/build.js",
-    "test": "test -f src/data/dependencies.json || node scripts/dependencies.js && node scripts/test.js"
+    "test": "test -f src/data/dependencies.json || node scripts/dependencies.js && node scripts/test.js",
+    "lint:fix:js": "prettier-eslint --write $(pwd)'/**/*.js'",
+    "lint:fix:css": "prettier --write $(pwd)'/**/*.css'",
+    "lint:fix:scss": "prettier --write $(pwd)'/**/*.scss'",
+    "lint:fix:json": "prettier --write $(pwd)'/**/*.json'",
+    "lint:fix": "npm run lint:fix:js && npm run lint:fix:css && npm run lint:fix:scss && npm run lint:fix:json",
```

Comment:
I guess we could simplify this into a single rule? I see no need for ever calling prettier only on a subset of files.

(Same for rule `lint` as well, of course.)

Reply:
Totally agree, cleaned it up.